beta/azure-container-apps/ReadMe.md

```diff
@@ -0,0 +1,228 @@
+   1. Click the “Upload/Download files” button and choose Upload.
+   2. Find the `scimsession` file that you saved to your computer and choose it.
+   3. Make note of the upload destination, then click Complete.
+7. Create Azure Container App Secret:
+    ```
+    az containerapp secret set -n $ContainerAppName -g $ResourceGroup --secrets scimsession="$(cat /home/$USER/scimsession | base64)"
+    ```
+8. Update your Container App with the required secondary container for Redis:
+    ```
+    az containerapp update -n $ContainerAppName -g $ResourceGroup --container-name op-scim-redis \
+    --image docker.io/redis --cpu 0.25 --memory 0.5Gi --set-env-vars 'REDIS_ARGS=--maxmemory 256mb --maxmemory-policy volatile-lru'
+    ```
+9. Update your Container App to allow your SCIM bridge container to use the secret you have created:
+    ```
+    az containerapp update -n $ContainerAppName -g $ResourceGroup --container-name op-scim-bridge \
+    --set-env-vars 'OP_REDIS_URL=redis://localhost:6379' 'OP_SESSION=secretref:scimsession'
+    ```
+10. Find your fdqn of your deployed application, this will be known as your SCIM bridge URL:
+    ```
+    az containerapp ingress show -n $ContainerAppName -g $ResourceGroup
+    ```
+11. Open the fqdn listed in a seperate browser tab, where you can test your connection. You should be prompted to log into the SCIM bridge with your `bearer token`.
+
+### Step 3: Follow the steps to connect your Identity provider to the SCIM bridge.
+ - [Connect your Identity Provider](https://support.1password.com/scim/#step-3-connect-your-identity-provider)
+
+
+<hr>
+
+## Manual Azure Portal Deployment Steps
+
+<details>
+<summary> Manual Azure Portal Deployment Steps</summary>
+
+### Step 1: Deploy your base SCIM bridge within Azure Container Apps
+
+1. First we create a new Container App from the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps).
+2. Either use an existing Resource Group or select Create New under the **Resource Group** Section.
+3. Enter the **Container app name**, for example *op-scim-con-app*.
+4. Select your **Region** for the deployment.
+5. Either use the existing Container App Environment generated or select Create New under the **Container Apps Environment** Section.
+    1. If creating a new Container App Environment, use a name that makes sense for your organization, for example op-scim-con-app-env.
+    2. Leave **Consumption** selected, and **Zone Redundancy** disabled.
+    3. Select **Create** on the create container apps environment window.
+6. Select **Next : Container >** to continue the container app creation.
+    1. Uncheck **Use quickstart image**.
+    2. Change the name of the container to *op-scim-bridge*.
+    3. Select **Docker Hub** for the image source.
+    4. Enter **1password/scim:v2.8.1** for the **Image and Tag**.
+    5. Leave the CPU and Memory as 0.25 CPU cores, 0.5 Gi memory.
```

Comment:
~~This is likely fine for "standard" deployments.~~
* ~~Do we know if this scales dynamically, vertically?~~
* ~~If not, should we make it clear that this resourcing is suitable for deployments of <1000 users?~~
I see you have a Todo at the bottom for exactly this. Disregard!

Reply:
Keeping this on the to-do for future improvements.